net.sf.eclipsecs.ui/src/net/sf/eclipsecs/ui/preferences/CheckstylePreferencePage.java

```diff
@@ -84,6 +89,15 @@
  */
 public class CheckstylePreferencePage extends PreferencePage implements IWorkbenchPreferencePage {
 
+  private static final String DEFAULT_LANGUAGE = "default";
+
+  private static final List<String> SUPPORTED_LANGUAGES = Stream.concat(
```

Comment:
The current code is meant to be fault-tolerant for future changes where someone just adds another language at the end, without thinking about the order. If I want something to be sorted, then I don't rely on it being sorted "by chance", but rather explicitly sort at every place where I want it to be sorted. Otherwise we create implicit dependencies between code pieces that are hard to recognize and maintain.